distribution/ddf-common/src/main/resources/etc/users.attributes

```diff
@@ -1,8 +1,10 @@
 {
     "admin" : {
-        "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress":"admin@localhost.local"
+        "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress":"admin@localhost.local",
```

Comment:
What happened to the `ssh` role?

Reply:
karaf doesn't know about users.attributes so it wouldn't work and on top of that I believe we want to remove it when securing a system anyway

Reply:
If we are adding these roles to `users.attributes`, can we move these roles out of `users.properties`-- 
``admin,manager,viewer,system-admin,systembundles` -- to avoid duplication?

Reply:
What we could do, in a future PR is just replace the "karaf" JAAS realm with one that would be aware of the attributes set in this file.